app/lib/api-wrappers/aurora/getUserLogbook.ts

```diff
@@ -0,0 +1,64 @@
+import { sql } from '@/lib/db'; // Assuming you have a database utility setup
```

Comment:
Didn't this file already exist 🤔

Reply:
yes it did exist but it only fetched the logbook for one single climb uuid and it did fetch bids (without difficulty).
I expanded the query in your getLogbook.ts to allow both query, depending if you pass an uuid or not. so your calls are still working as expected.
a) if a climbuuid is passed you get the logbook as before
b) if no climbuuid is passed you get a logbook where climbs without difficulties are removed from the dataset 
--
but depending on the stats we want to show, we may need to add the bids too